src/dsp_tools/commands/start_stack.py

```diff
@@ -90,12 +89,16 @@ def _get_url_prefix(self) -> str:
         """
         url_prefix_base = "https://github.com/dasch-swiss/dsp-api/raw"
 
+        docker_compose_pth = importlib.resources.files("dsp_tools").joinpath("resources/start-stack/docker-compose.yml")
+        docker_compose = yaml.safe_load(docker_compose_pth.read_bytes())
+        tag = docker_compose["services"]["api"]["image"].split(":")[-1]
+
+        response = requests.get(f"https://api.github.com/repos/dasch-swiss/dsp-api/git/ref/tags/{tag}", timeout=5)
+        commit_of_used_api_version = response.json()["object"]["sha"]
```

Comment:
Question: Maybe I'm missing something here. But from my understanding, you're getting the SHA only to construct raw links.  
But GitHub allows raw links to be constructed from tags  
(e.g. `https://raw.githubusercontent.com/dasch-swiss/dsp-api/refs/tags/v31.4.0/webapi/scripts/fuseki-repository-config.ttl.template` )  
So couldn't you simply use the tag from the docker-compose.yml to construct the links directly?

Reply:
The one who missed something is me :-) Thanks, this simplifies a lot